doc/how-to/initialise.rst

```diff
@@ -90,6 +90,37 @@ Complete the following steps to initialise MicroCloud:
 
 See an example of the full initialisation process in the :ref:`Get started with MicroCloud <initialisation-process>` tutorial.
 
+Excluding MicroCeph or MicroOVN from MicroCloud
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If the MicroOVN or MicroCeph snap is not installed on the system that runs :command:`microcloud init`, you will be prompted with the following question:
+
+     MicroCeph not found. Continue anyway? (yes/no) [default=yes]:
+
+     MicroOVN not found. Continue anyway? (yes/no) [default=yes]:
+
+  If you choose ``yes``,  only existing services will be configured on all systems.
+
+  If you choose ``no`` the setup will be cancelled.
```

Comment:
```suggestion
    MicroCeph not found. Continue anyway? (yes/no) [default=yes]:

    MicroOVN not found. Continue anyway? (yes/no) [default=yes]:

If you choose ``yes``,  only existing services will be configured on all systems.
If you choose ``no``, the setup will be cancelled.
```

Too much indentation (must be 4 spaces afaik).
Also, missing comma in the last sentence, and I would combine the last two sentences into one paragraph (which means get rid of the empty line between them).